components/MDX/Calendar/Event/index.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+import type { ComponentProps } from 'react';
+
+import Event from '@/components/MDX/Calendar/Event';
+
+type Story = StoryObj<typeof Event>;
+type Meta = MetaObj<typeof Event>;
+
+const defaultArgs: ComponentProps<typeof Event> = {
+  start: {
+    date: new Date().toISOString(),
+  },
+  end: {
+    date: new Date().toISOString(),
```

Comment:
Can you not use Date objects? Otherwise this date will change every-day, rending the visual testing useless as the date changes every day.